src/vivarium/framework/population/manager.py

```diff
@@ -188,7 +213,7 @@ def setup(self, builder: Builder) -> None:
             self.register_simulant_initializer, allow_during=[lifecycle_states.SETUP]
         )
 
-        self.register_simulant_initializer(self, creates_columns=self.columns_created)
+        self.register_simulant_initializer(self)
```

Comment:
Pretty sure we don't need to do this anymore, since the pop manager no longer creates the tracked column